src/main/java/edu/harvard/iq/dataverse/api/Access.java

```diff
@@ -933,7 +934,7 @@ private boolean isAccessAuthorized(DataFile df, String apiToken) {
         if (session != null) {
             if (session.getUser() != null) {
                 if (session.getUser().isAuthenticated()) {
-                    user = (AuthenticatedUser) session.getUser();
+                    user = session.getUser();
```

Comment:
If we don't need to cast to `AuthenticatedUser` why are we doing `getUser().isAuthenticated()`?